core/integration/features/verified/risk-parameters-sigma-test.feature

```diff
@@ -0,0 +1,165 @@
+      | market.stake.target.scalingFactor             | 1     |
+      | market.liquidity.bondPenaltyParameter         | 0.2   |
+      | market.liquidity.targetstake.triggering.ratio | 0.1   |
+
+    And the average block duration is "1"
+
+    And the parties submit the following liquidity provision:
+      | id  | party  | market id | commitment amount | fee   | side | pegged reference | proportion | offset | lp type    |
+      | lp1 | party0 | ETH/MAR0  | 10000000          | 0.001 | sell | ASK              | 500        | 20     | submission |
+      | lp1 | party0 | ETH/MAR0  | 10000000          | 0.001 | buy  | BID              | 500        | 20     | amendment  |
+
+    And the parties place the following orders:
+      | party  | market id | side | volume | price | resulting trades | type       | tif     | reference   |
+      | party1 | ETH/MAR0  | buy  | 10     | 90    | 0                | TYPE_LIMIT | TIF_GTC | buy-ref-11  |
+      | party1 | ETH/MAR0  | buy  | 1      | 100   | 0                | TYPE_LIMIT | TIF_GTC | buy-ref-12  |
+      | party2 | ETH/MAR0  | sell | 1      | 100   | 0                | TYPE_LIMIT | TIF_GTC | sell-ref-13 |
+      | party2 | ETH/MAR0  | sell | 10     | 110   | 0                | TYPE_LIMIT | TIF_GTC | sell-ref-14 |
+
+    When the opening auction period ends for market "ETH/MAR0"
+    And the trading mode should be "TRADING_MODE_CONTINUOUS" for the market "ETH/MAR0"
+
+    And the market data for the market "ETH/MAR0" should be:
+      | mark price | trading mode            | horizon | min bound | max bound | target stake | supplied stake | open interest |
+      | 100        | TRADING_MODE_CONTINUOUS | 43200   | 91        | 109       | 355          | 10000000       | 1             |
+    # target_stake = mark_price x max_oi x target_stake_scaling_factor x rf_short = 1 x 100 x 1 x 3.5569036 =355
+
+    Then the order book should have the following volumes for market "ETH/MAR0":
+      | side | price | volume |
+      | sell | 110   | 181829 |
+      | sell | 109   | 0      |
+      | buy  | 91    | 0      |
+      | buy  | 90    | 222233 |
+
+    And the parties should have the following account balances:
+      | party  | asset | market id | margin   | general                     | bond     |
+      | party0 | USD   | ETH/MAR0  | 97006899 | 499999999999999999892993101 | 10000000 |
+      | party1 | USD   | ETH/MAR0  | 1336     | 4999998664                  | 0        |
+      | party2 | USD   | ETH/MAR0  | 5884     | 4999994116                  | 0        |
+
+    # mentainance margin level for LP: 181819*100*3.5569036=6.47e7
+    # initial  margin level for LP: 181819*100*3.5569036 *1.2=9.7e7
+
+    And the parties should have the following margin levels:
+      | party  | market id | maintenance | search   | initial  | release   |
+      | party0 | ETH/MAR0  | 64671266    | 77605519 | 97006899 | 109941152 |
+
+# Scenario: 003, test market ETH/MAR54(sigma=100),
```

Comment:
is that commented in purpose?

Reply:
yes, I got the following error message when sigma is greater than 52, so I am going to change the validated range of sigma from 100 to 50 

2022-11-02T15:52:30.600Z	INFO	execution	price/statevar.go:58	no reference price available for market - cannot calculate price ranges	{"event-id": "20191130_000002"}
the opening auction period ends for market "ETH/MAR54" FATAL: some mock assertion failed: Cannot create a Decimal from NaN